### PR TITLE
chore: doc to include spinner

### PR DESCRIPTION
**Pull request by nightnei**

### CODE REVIEW

File: docs/ilc_app_interface.md [hunk 1]
```diff
@@ -59,4 +59,14 @@ See more information about the [lifecycle functions here](https://single-spa.js.
 * `domElementGetter(): HTMLElement` - returns ref to `HTMLElement` that should be used as container to render app's content
 * `getCurrentPathProps(): {}` - returns _Props_ that were defined for current path
 * `getCurrentBasePath(): string` - returns same value as `basePath` param in `routerProps` query parameter
-* `errorHandler(error, errorInfo = {}): void` - app MUST use it to propagate all unhandled errors
\ No newline at end of file
+* `errorHandler(error, errorInfo = {}): void` - app MUST use it to propagate all unhandled errors
+
+### Page transitions spinner
+You can set own spinner with help of your main html-template:
```
Comment (oleh-momot) [inline docs/ilc_app_interface.md:65]: with *the* help of

Comment (StyleT) [reply docs/ilc_app_interface.md:65]: - I think we need to move it to separate doc called "Animation during reroute" 
- This particular example will never work as is. Pls give a concrete one that users can copy/paste.

Comment (oleh-momot) [reply docs/ilc_app_interface.md:65]: https://github.com/namecheap/ilc/pull/143/commits/1606d29992c67cff54850a95c2b4face8fbfb99e

File: docs/animation_during_reroute.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Animation during reroute
+
```
Comment (StyleT) [inline docs/animation_during_reroute.md:2]: - add example animation
- more info about logic of the spinner appearance

Comment (StyleT) [reply docs/animation_during_reroute.md:2]: - Rm clone of the DOM nodes in we have no spinner
- Mention importance of the overlay

**Pull request merged**